operation/operation.c

```diff
@@ -123,7 +89,6 @@ void logf_msg(enum logs logEnum, oper_st * state, int n_args, ...) {
 
     printf("%s\n", msg);
     write_log_entry(msg, &state->eeprom_log_idx);
-    write_to_eeprom(LOG_INDEX_ADDR, &state->eeprom_log_idx, 1);
```

Comment:
line 126 is removed since write_log_entry() function do store the log index to EEPROM, no need to call it again here